File: SoloLearn Codes/Ballpark Orders/BallparkOrders.java
import java.util.*;

public class BallparkOrders
{
    public static void main(String[] args) {
        String[] orders = new Scanner(System.in).nextLine().split(" ");
        Map<String, Double> menu = Map.of(
                "Nachos", 6.00, "Pizza", 6.00,
                "Cheeseburger", 10.00, "Water", 4.00,
                "Coke", 5.00);
        double total = Arrays.stream(orders)
                .mapToDouble(c -> menu.containsKey(c) ? menu.get(c) : menu.get("Coke"))
                .sum();
        System.out.println(Math.round((total * 1.07) * 100.0) / 100.0);
    }
}

File: SoloLearn Codes/Ballpark Orders/ballpark_orders.rb
hash = { "Nachos"=> 6, "Pizza"=> 6, "Cheeseburger"=> 10, "Water"=> 4, "Coke"=> 5 }
cost = gets.chomp.split(" ").map{ |c| hash.key?(c) ? hash[c] : hash["Coke"] }.sum
print cost + cost * 0.07